radixdlt/src/integration/java/com/radixdlt/consensus/simulation/network/SimulationNodes.java

```diff
@@ -105,9 +109,12 @@ public SimulationNodes(
 
 	private ConsensusRunner createBFTInstance(ECKeyPair key) {
 		final Mempool mempool = new EmptyMempool();
-		final Hasher hasher = new DefaultHasher();
+		final Hasher nullHasher = o -> Hash.ZERO_HASH;
+		final Hasher defaultHasher = new DefaultHasher();
+		final HashSigner nullSigner = (k, h) -> new ECDSASignature();
+		final HashVerifier nullVerifier = (p, h, s) -> true;
```

Comment:
I think mocked hashers/signers are fine for Travis runs but I think we should be running real hashers and signers for Jenkins no?

Just a comment, no need to change anything for now

Reply:
I think null hashers/signers are fine until we have adversaries which sign incorrectly.